io.openems.edge.io.iot2000/src/io/openems/edge/io/iot2000/api/IOT2000Util.java

```diff
@@ -0,0 +1,253 @@
+	
+	private static final IOT2000Util OBJ = new IOT2000Util();
+	
+	private IOT2000Util() {
+		setupPins();
+	}
+	
+	public static IOT2000Util getInstance() {
+		return OBJ;
+	}
+
+	private int setupPins() {
+
+		int status = 0;
+
+		for (String gpio : ALLPINS) {
+			try {
+				unexport(gpio);
+			} catch (IOException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+				status -= 1;
+			}
+
+			try {
+				export(gpio);
+			} catch (IOException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+				status -= 1;
+			}
+
+		}
+
+		if (status != 0) {
+			return status;
+		}
+
+		// LED
+		try {
+			setDirLow("46");
+			setDirLow("30");
+			setDirIn("31");
+			setVal("46", '0');
+			setVal("30", '0');
+			setDirLow("7");
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			status++;
+		}
+
+		// DI0
+		try {
+			setDirIn("15");
+			setDirHigh("42");
+			setDirIn("43");
+			setVal("42", '1');
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			status++;
+		}
+
+		// DI1
+		try {
+			setDirIn("5");
+			setDirLow("44");
+			setDirLow("72");
+			setDirHigh("24");
+			setDirIn("25");
+			setVal("24", '1');
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			status++;
+		}
+
+		// DI2
+		try {
+			setDirIn("10");
+			setDirLow("74");
+			setDirHigh("26");
+			setDirIn("25");
+			setVal("26", '1');
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			status++;
+		}
+
+		// DI3
+		try {
+			setDirIn("4");
+			setDirLow("70");
+			setDirHigh("22");
+			setDirIn("23");
+			setVal("22", '1');
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			status++;
+		}
+
+		// DI4
+		try {
+			setDirIn("6");
+			setDirHigh("36");
+			setDirIn("37");
+			setVal("36", '1');
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			status++;
+		}
+
+		// DQ0
+		try {
+			setDirLow("40");
+			setDirIn("41");
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			status++;
+		}
+
+		// DQ1
+		try {
+			setDirLow("38");
+			setDirIn("39");
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			status++;
+		}
+
+		return status;
+
+	}
+
+	private void unexport(String gpio) throws IOException {
+
+		byte[] b = gpio.getBytes();
+		File file = new File("/sys/class/gpio/unexport");
+		FileOutputStream fos = new FileOutputStream(file);
+		fos.write(b);
+		fos.close();
+	}
+
+	private void export(String gpio) throws IOException {
+		byte[] b = gpio.getBytes();
+		File file = new File("/sys/class/gpio/export");
+		FileOutputStream fos = new FileOutputStream(file);
+		fos.write(b);
+		fos.close();
+	}
+
+	private void setDirHigh(String gpio) throws IOException {
```

Comment:
maybe setDirectionHigh instaed if setDirHigh and so on for all those setter functions